src/Controller/TestController.php

```diff
@@ -1,23 +1,21 @@
 <?php
```

Comment:
Can we keep this controller around? For me (or a Ready member) it is a convenient and reliable way to verify the credentials are correct. I think we should remove it once we have built the validation into admin settings form's submit validation.